features/steps/header.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from behave import given, then
+
+from docx import Document
+from docx.header import Header
+
+from helpers import test_docx
+
+
+# given ===================================================
+
+@given('a section')
+def given_a_section(context):
+    document = Document(test_docx('doc-default'))
+    context.section = document.sections[0]
```

Comment:
* This one should go in the section.py steps file. The 'Then section.header is...' one below too.
* I've left out the other two for now. I think we'll have our hands full with this one for half-a-dozen commits believe it or not. It will drive implementation of HeaderPart and a number of other dependencies.
* Any new test files required (like 'doc-default.docx') need to be included in the commit (in features/steps/test_files/). These acceptance tests need to run to an error driven by missing implementation. That means all the imports need to work and the test file loads as well (and syntax etc.). I've changed this to use an existing test file because any section will do, so no new file this time around. New files should start with the three-letter abbreviation of the main object under test, generally the same one used by the steps file, so 'sct-' in this case and 'hdr-' for most of what we do later.
* A bit further to that point, you will need to be running these tests before submission. Firstly, you'll need to work out any errors in the test code itself. Second, you'll need the "actual" error it produces to know what to implement next; Section.header in this case because the error is 'AttributeError: 'Section' object has no attribute 'header''. You'll want to be seeing that error for yourself each time.